### DISCUSSION TITLE
可以使用blender把mmd模型提取出来

### DISCUSSION THREAD

**Discussion by MrChenLearnSpace** [stalomeow/StarRailNPRShader#3] [Ideas]
可以使用blender把mmd模型提取出来，后面操作，做动作做动作及布料渲染都可以直接在blender中烘好，这样整个demo会更丰满

**Comment by stalomeow** [comment]
我这个项目做的是实时渲染，不是mmd，懒得做这些东西。布料之类的，我一般都在unity里用 MagicaCloth 做。
